### ISSUE TITLE
About the results of SASRec model

### ISSUE THREAD

**Issue by FelixHilyer** [aHuiWang/CIKM2020-S3Rec#11]
Hi, thanks for your great work!
But I have a question. I reran the source code of the SASRec model, using multiple 5-core datasets you provided. But I found that their results are different from the results you reported.  For example，i use the code of https://github.com/pmixer/SASRec.pytorch, and the Beauty.txt file you provided in the data folder, we will get the result NDCG@10: 0.3384 and HR@10: 0.5059. Besides, on the sports dataset, we can also get the results NDCG@10: 0.3139 and HR@10: 0.5058.  At the same time, we modified the code ourselves so that SASRec does not negatively sample and sorts on all items, and the results are far from the data you provided. Can you provide some instructions on how to get the performance results of the SASRec model on the 5-core dataset?
Thanks

**Comment by aHuiWang** [comment]
Sorry for the late reply. 

I checked the code and found that the default setting head_num is 1 in your repo, the loss function uses L2 Norm, and the feedforward network is implemented using nn.conv1d. I am not sure whether these will lead to better results. Our experiments put more attention on verifying the effectiveness of pre-training. For a fair comparison, we used the same training settings as SASRec in the fine-tuning phase.

Regarding the full sorting results, I used the optimal parameters of the sampling training and ran the full sorting directly, so if you don’t sample, you will definitely get a result that is much better than ours
